### PR TITLE
Add blob fees to Rollup Economics

### PR DESCRIPTION
**Pull request by Jam516**
Track how much rollups are spending on submitting blobs

### CODE REVIEW

**Review by 0xRobin — COMMENTED** [review]
couple small comments. 
I was also thinking, maybe we should include a `l1_fees` model that brings all the data, verification and blob fees together (on a daily level so it's very complimentary to `l2_revenue`).
I'd include columns for each fee type and then 1 general fee column summing all of them together.

**Comment by Jam516** [comment]
> couple small comments. I was also thinking, maybe we should include a `l1_fees` model that brings all the data, verification and blob fees together (on a daily level so it's very complimentary to `l2_revenue`). I'd include columns for each fee type and then 1 general fee column summing all of them together.

Hey @0xRobin , added in the l1_fees model. I think its a great idea! And resolved the comments

File: models/rollup_economics/ethereum/l1_blob_fees.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+AND p.symbol = 'ETH'
+AND p.minute >= timestamp '2024-03-13'
+AND s.block_time >= timestamp '2024-03-13'
+{% if is_incremental() %}
+AND s.block_time >= date_trunc('day', now() - interval '7' day)
+AND p.minute >= date_trunc('day', now() - interval '7' day)
+{% endif %}
+AND s.blob_submitter_label IN ('Arbitrum',
+'Linea',
+'zkSync',
```
[the remark this answers is not in the document]
Comment (Jam516) [reply models/rollup_economics/ethereum/l1_blob_fees.sql]: done

File: models/rollup_economics/ethereum/l1_blob_fees.sql [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{{ config(
+    schema = 'rollup_economics_ethereum',
+    alias = 'l1_data_fees',
```
[the remark this answers is not in the document]
Comment (Jam516) [reply models/rollup_economics/ethereum/l1_blob_fees.sql]: done

File: models/rollup_economics/ethereum/l1_blob_fees.sql [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    unique_key = ['name', 'hash'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "rollup_economics",
+                                    \'["niftytable"]\') }}'
+)}}
+
+SELECT 
+lower(blob_submitter_label) as name,
+tx_hash as hash,
```
[the remark this answers is not in the document]
Comment (Jam516) [reply models/rollup_economics/ethereum/l1_blob_fees.sql:16]: left

**Review by 0xRobin — APPROVED** [review]
great, ship it! :rocket:

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**